lib/features/steps/runner_steps.rb

```diff
@@ -64,6 +64,23 @@
   Docker.start_service(service, command: one_line_cmd)
 end
 
+# Allows validation of the last exit code of the last run docker-compose command.
+# Will fail if no commands have been run.
+#
+# @step_input expected_code [Integer] The expected exit code
+Then('the exit code of the last docker command was {int}') do |expected_code|
+  exit_code = Docker.last_exit_code
+  assert_not_nil(exit_code, 'No docker exit code available to verify')
+  asset_equal(exit_code, expected_code)
```

Comment:
Missing `r` in assert, but also the expected code should be the first argument.  See also L81.